agile-development/team-agreements/team-manifesto/readme.md

```diff
@@ -0,0 +1,78 @@
+
+## How to Build a Team Manifesto
+
+It can be said that the best time to start building it is at the very early phase of the engagement when teams meet with each other for swarming or during the preparation phase.
+
+It is recommended to keep team manifesto as simple as possible, so preferably, one-page simple document which **doesn't include any references or links** is a nice format for it.
+If there is a need for providing knowledge on certain topics, the way to do is delivering brown-bag sessions, technical katas, practices documentations and others later on.
+
+A few important points about team manifesto;
+
+- Team manifesto is built by the **team itself**
+
+- It can include any technical or behavioural agility topics as soon as team find it relevant,
+
+- It aims to have a common understanding about the desired expertise, practices and/or behaviours within the team
+
+- Based on the needs of the team and retrospective results, it can be modified during the engagement.
+
+In Microsoft, we have certain standards to build high quality and well-crafted software as well as to give comfortable/transparent environment to each team member for helping her/him to reach her/his highest potential.
+
+The difference of team manifesto from other comprehensive set of standards/practices documents is, it is used to give a short summary of the upcoming way of working and practices to prepare a base before deep dive starts with open discussion environment in an ice-breaker way.
+
+Below, you can find some including, but not limited, topics many teams touch during engagements,
+
+| Topic | What is it about ? |
+|-|-|
+| Collective Ownership | Does team own the code rather than individuals? And what is the expectation? |
+| Respect | Any preferred statement about it's a "must-have" team value |
+| Collaboration | Any preferred statement about how does team want to collaborate ? |
+| Transparency | A simple statement about it's a "must-have" team value and if preferred, how does this being provided by the team ? meetings, retrospective, feedback mechanisms ..etc. |
+| Craftpersonship | Which tools such as Git, VS Code LiveShare, ..etc. are being used ? What is the definition of expected best usage of them? |
+| PR sizing | What does team prefer in PRs ? |
+| Branching | Team's branching strategy and standards |
+| Commit standards | Preferred format in commit messages, rules and more |
+| Clean Code | Does team follow clean code principles ? |
+| Pair/Mob Programming | Will team apply pair/mob programming ? If yes, what programming styles are suitable for the team ? |
+| Release Process | Principles around release proicess such as quality gates, reviewing process ...etc. |
+| Code Review | Any rule for code reviewing such as min number of reviewers, team rules ...etc. |
+| Action Readiness | How the backlog will be groomed ? What will be the way for clear DoD and ACs ? |
```

Comment:
"How the backlog will be refined?"

Reply:
fixed commit b07b8657c0ea24e73211d9bfc93994abe9cf3545